bin/nvcc_wrapper

```diff
@@ -100,7 +100,7 @@ depfile_target_arg=""
 remove_duplicate_link_files=0
 
 function warn_std_flag() {
-  echo "nvcc_wrapper - *warning* you have set multiple optimization flags (-std=c++1* or --std=c++1*), only the first is used because nvcc can only accept a single std setting"
+  echo "nvcc_wrapper - *warning* you have set multiple standard flags (-std=c++1* or --std=c++1*), only the last is used because nvcc can only accept a single std setting"
```

Comment:
This line is different from `develop`, see https://github.com/kokkos/kokkos/blob/9208fb959c6fc93d7ce727b7b6082a0cfabd5122/bin/nvcc_wrapper#L103.